README.md

```diff
@@ -26,23 +26,86 @@ $ php composer.phar require akeneo/api-php-client php-http/guzzle6-adapter
 
 If you want to use another HTTP client implementation, you can check [here](https://packagist.org/providers/php-http/client-implementation) the full list of HTTP client implementations. 
 
-# Getting started
+## Getting started
+
+### Initialise the client
+You first need to initialise the client with your credentials client id/secret and with your user/password.
+
+If you don't have any client id, let's a look at [this page](https://api.akeneo.com/documentation/security.html#authentication) to create it.
 
 ```php
 <?php
 
-require_once '/vendor/autoload.php';
+require_once __DIR__ . '/vendor/autoload.php';
 
 $clientBuilder = new \Akeneo\Pim\Client\AkeneoPimClientBuilder('http://localhost/');
 $client = $clientBuilder->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+```
+
+You can authenticate to the client with your token/refresh token as well.
+```php
+<?php
 
-$categories = $client->getCategoryApi()->getCategories();
+require_once __DIR__ . '/vendor/autoload.php';
+
+$client = $clientBuilder->buildAuthenticatedByToken('client_id', 'secret', 'token', 'refresh_token');
+```
+
+Getting the token and refresh token is as simple as:
+```php
+$client->getToken();
+$client->getRefreshToken();
+```
+
+### Get a product
+
+```php
+$product = $client->getProductApi()->get('top');
+echo $product['identifier']; // display "top"
+```
 
-foreach ($categories->getItems() as $category) {
-    //...
+### Get a list of products
+
+```php
+$searchBuilder = new SearchBuilder();
+$searchBuilder->addFilter('enabled', '=', true);
+$searchFilters = $searchBuilder->getFilters();
+
+$products = $client->getProductApi()->all(50, ['search' => $searchFilters]);
```

Comment:
I didn't want to put too much in the getting started, but yes if you think it's necessary.